test/integrations/validation/Zod.spec.ts

```diff
@@ -0,0 +1,348 @@
+
+import { expect, describe, it } from 'vitest';
+
+import zod from '../../../src/integrations/validation/module';
+
+import { SCHEMAS, ERRORS } from './_fixtures/Zod.fixtures';
+
+describe('Zod', () =>
+{
+    describe('should validate using the String schema', () =>
+    {
+        it('should validate name', () =>
```

Comment:
The description `should validate name` doesnt tell me what goes wrong if it fails. It only tells what part of the fixture you're using. Suggestion: `should accept a valid value`.